pandas_vs_excel/pandas_vs_excel.ipynb

```diff
@@ -0,0 +1,640 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 79,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style>\n",
+       "    .dataframe thead tr:only-child th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: left;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Student Name</th>\n",
+       "      <th>Exam 1 Score</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>Edwin Duncan</td>\n",
+       "      <td>83</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>Steve Hayes</td>\n",
+       "      <td>69</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>Cecilia Richards</td>\n",
+       "      <td>91</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>Dorothy Harris</td>\n",
+       "      <td>53</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>Jordan Davis</td>\n",
+       "      <td>76</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "       Student Name  Exam 1 Score\n",
+       "0      Edwin Duncan            83\n",
+       "1       Steve Hayes            69\n",
+       "2  Cecilia Richards            91\n",
+       "3    Dorothy Harris            53\n",
+       "4      Jordan Davis            76"
+      ]
+     },
+     "execution_count": 79,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# import pandas library\n",
+    "import pandas as pd\n",
+    "\n",
+    "# create pandas DataFrame from CSV\n",
+    "exam_one = pd.read_csv('csv/exam_one.csv', delimiter=',')\n",
+    "exam_two = pd.read_csv('csv/exam_two.csv', delimiter=',')\n",
+    "sections = pd.read_csv('csv/sections.csv', delimiter=',')\n",
+    "\n",
+    "# print the top five entires of the DataFrame\n",
+    "exam_one.head()"
```

Comment:
I'm wondering if it's also worth printing out sections.head() too? Up to you though.

Reply:
I'll keep it as is.